cqc/pythonLib.py

```diff
@@ -2379,3 +2471,352 @@ def getTime(self, block=True):
             return otherHdr.datetime
         except AttributeError:
             return None
+
+
+class CQCToFile(CQCHandler):
+    """Handler to be used when writing the CQC commands to a file."""
+
+    def __init__(self, filename='CQC_File', pend_messages=False,
```

Comment:
I think we should also provide the option to specify the binary file path. Or maybe there can be an argument for if we should output text or binary and there is only one file. Also, I think the argument should instead be filepath such that the user can specify a file in another folder.

Reply:
I have added an argument where you choose to have binary output or text, so there's only one file.

Still to do: absolute path instead of relative.

Reply:
Done 8e0e6072704febffd5b4dc56a51a95691079ed1a